mkdocs.yml

```
@@ -38,6 +38,7 @@ nav:
     - Icons:                              'fundamentals/icons.md'
     - Localization:                       'fundamentals/localization.md'
     - Styles:
+      - Borders:                           'fundamentals/styles/borders.md'
```

YAMLlint (1.37.1)

[error] too many spaces after colon

(colons)